src/Oscardb.jl

```diff
@@ -0,0 +1,232 @@
+module Oscardb
```

Comment:
This really should be camel case (and the filename then should match).
```suggestion
module OscarDB
```
But the "Oscar" it also feels a bit redundant. Why isn't this module called `DB` or `Database` or `Databases` or so?